extensions/eclipse-che-theia-workspace/package.json

```diff
@@ -18,7 +18,8 @@
     "@eclipse-che/api": "latest",
     "@theia/workspace": "next",
     "@eclipse-che/theia-remote-api": "^0.0.1",
-    "js-yaml": "3.13.1"
+    "js-yaml": "3.13.1",
+    "react": "^16.8.0"
```

Comment:
why react there ?

Reply:
If you execute `yarn run lint:fix`, you will have
```
/home/vitaliy/projects/devworkspace-install-plugins/che-theia-backup-2/extensions/eclipse-che-theia-workspace/src/browser/che-navigator-widget.tsx
  11:24  error  'react' should be listed in the project's dependencies. Run 'npm i -S react' to add it  import/no-extraneous-dependencies
```

It's reasonable, as the react is used in this package.
https://github.com/eclipse-che/che-theia/blob/main/extensions/eclipse-che-theia-workspace/src/browser/che-navigator-widget.tsx#L11

The same for other extension, where I added missing dependencies (react, phosphor,..).